### PR TITLE
Shadowrun6th-German - Add french translation

### PR DESCRIPTION
**Pull request by rpelisse**
## Changes / Comments

Just add the french translation for the Characters Sheet. A lot of thanks to character sheet creator for this sheet!

## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [X] Does the pull request title have the sheet name(s)? Include each sheet name.
- [ ] Is this a bug fix? (no)
- [ ] Does this add functional enhancements (new features or extending existing features) ? (no)
- [ ] Does this add or change functional aesthetics (such as layout or color scheme) ?  (no)
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

### CODE REVIEW

File: Shadowrun6th-German/translations/fr.json [hunk 1]
```diff
@@ -1,364 +1,364 @@
...
+	"mod_conjuring": "Modificateurs de conjuration",
+	"mod_magic": "Modificateurs de magie",
+	"mod_spells": "Modificateurs de sorcellerie",
+	"monitor_modifier_phys": "Modificateur de moniteur de condition physiques",
+	"monitor_modifier_stun": "Modificateur de moniteur de condition d'étourdissement",
+	"noedge": "Tu n'as plus de Chance/Atout.",
+	"notes": "Notes",
 	"offense": "Angriffswurf",
 	"phys": "körperlich",
 	"phys_short": "K",
```
Comment (Bltgn) [inline Shadowrun6th-German/translations/fr.json:224]: If that's the short for physical damage we need a P.

File: Shadowrun6th-German/translations/fr.json [hunk 2]
```diff
@@ -1,364 +1,364 @@
...
-	"9": "9W6",
-	"act_banishing": "Verbannung eines Geistes",
-	"act_spellcasting": "Wirken eines Zaubers",
-	"act_spirit": "Handlung eines Geistes",
+	"8": "8d6",
+	"9": "9d6",
+	"act_banishing": "Bannir un esprit",
+	"act_spellcasting": "Sorcellerie",
+	"act_spirit": "Conjuration d'esprit",
 	"act_summoning": "Herbeirufung eines Geistes",
```
Comment (Bltgn) [inline Shadowrun6th-German/translations/fr.json:47]: That should be translated as well, is that for spirit summoning ?

File: Shadowrun6th-German/translations/fr.json [hunk 3]
```diff
@@ -1,364 +1,364 @@
...
-	"machinepistol": "Automatikpistole",
+	"machinegun": "Mitrailette",
+	"machinepistol": "Pistolet Automatique",
 	"magic": "Magie",
 	"magic_short": "MAG",
-	"man": "Mensch",
-	"manipulation_air": "Manip. (Luft)",
+	"man": "Homme",
+	"manipulation_air": "Manip. (Air)",
 	"manipulation_beast": "Manip. (Tier)",
```
Comment (Bltgn) [inline Shadowrun6th-German/translations/fr.json:203]: Not sure if Beast and Blood are used at the current state of 6e but translating that should make it future proof.

File: Shadowrun6th-German/translations/fr.json (hunk 1 above)
Comment (rpelisse) [reply Shadowrun6th-German/translations/fr.json]: @Bltgn nice catch! It's fixed now :)

**Comment by rpelisse** [comment]
@clevett Cassie, you may be interested in this one :) (even if I know you prefer Anarchy ;) )

**Comment by clevett** [comment]
@rpelisse Thanks for the ping. Translations are best done with Crowdin so prevent any upload conflicts. If you email me at  cassie@roll20.net then I can get you an invite. Please include the languages you want to translate in. Thanks. :)

**Pull request merged**